skyplane/cli/cli_impl/cp_replicate.py

```diff
@@ -127,22 +127,25 @@ def generate_full_transferobjlist(
     dest_bucket: str,
     dest_prefix: str,
     recursive: bool = False,
+    requester_pays: str = None
 ) -> List[Tuple[ObjectStoreObject, ObjectStoreObject]]:
     """Query source region and destination region buckets and return list of objects to transfer."""
     source_iface = ObjectStoreInterface.create(source_region, source_bucket)
     dest_iface = ObjectStoreInterface.create(dest_region, dest_bucket)
 
+    if requester_pays == 'requester' and isinstance(source_iface, S3Interface):
+        source_iface.requester_pays = True
```

Comment:
Actually, add a method like `set_requester_pays()` to `ObjectStoreInterface` with abstract methods (silently do nothing for GCP and Azure).